tests/network/user_defined_network/conftest.py

```
@@ -164,6 +167,35 @@ def udn_pod(
         client=admin_client,
+        label=ALLOWED_POD_LABEL,
+    ) as pod:
+        pod.wait_for_status(status=Pod.Status.RUNNING)
+        yield pod
+
+
+@pytest.fixture()
+def denied_pod(
+    admin_client: DynamicClient,
+    udn_namespace: Namespace,
+    namespaced_layer2_user_defined_network: Layer2UserDefinedNetwork,
```

Ruff (0.16.1)

[warning] Unused function argument: `namespaced_layer2_user_defined_network`

(ARG001)